install.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+set -e
+
+# Handy env vars.
+hw_url='hunterwittenborn.com'
+color_normal="$(tput sgr0)"
+color_green="$(tput setaf 77)"
+color_orange="$(tput setaf 214)"
+color_blue="$(tput setaf 14)"
+color_red="$(tput setaf 202)"
+color_purple="$(tput setaf 135)"
+
+# Answer vars.
+SETUP_PREBUILT_MPR=0
+MAKEDEB_PKG='makedeb'
```

Comment:
```suggestion
readonly MAKEDEB_PKG='makedeb'
```
Since it's always the same package